### PR TITLE
Azure Synapse fixes

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Fixes to the Azure Synapse doc as pointed by @saramashfej [here](https://github.com/rudderlabs/rudderstack-docs/pull/600#discussion_r884949517).

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

## Notion ticket link

> [Azure Synapse fixes](https://www.notion.so/rudderstacks/Azure-Synapse-fixes-ef55df7682754df2b730fb46d5cd6b6c)

### CODE REVIEW

File: docs/data-warehouse-integrations/azure-synapse.mdx [hunk 1]
```diff
@@ -20,9 +20,17 @@ Refer to the <a href="https://rudderstack.com/docs/data-warehouse-integrations/w
 Find the open-source transformer code for this destination in the <a href="https://github.com/rudderlabs/rudder-transformer/tree/master/v0/destinations/azure_synapse">GitHub repository</a>.
 </div>
 
-## Setting user permissions in Azure Synapse
+## Prerequisites
```
Comment (ameypv-rudder) [inline docs/data-warehouse-integrations/azure-synapse.mdx:23]:
```suggestion
## Pre-requisites
```

File: docs/data-warehouse-integrations/azure-synapse.mdx [hunk 2]
```diff
@@ -20,9 +20,17 @@ Refer to the <a href="https://rudderstack.com/docs/data-warehouse-integrations/w
 Find the open-source transformer code for this destination in the <a href="https://github.com/rudderlabs/rudder-transformer/tree/master/v0/destinations/azure_synapse">GitHub repository</a>.
 </div>
 
-## Setting user permissions in Azure Synapse
+## Pre-requisites
```
Comment (ameypv-rudder) [inline docs/data-warehouse-integrations/azure-synapse.mdx:23]:
```suggestion
## Prerequisites
```

File: docs/data-warehouse-integrations/azure-synapse.mdx [hunk 3]
```diff
@@ -20,9 +20,17 @@ Refer to the <a href="https://rudderstack.com/docs/data-warehouse-integrations/w
 Find the open-source transformer code for this destination in the <a href="https://github.com/rudderlabs/rudder-transformer/tree/master/v0/destinations/azure_synapse">GitHub repository</a>.
 </div>
 
-## Setting user permissions in Azure Synapse
+## Prerequisites
 
-To set up an Azure Synapse Analytics user, you require an Azure subscription for the SQL Data Warehouse. Additionally, you need to [create a schema (dedicated SQL pool)](https://docs.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/create-data-warehouse-portal) in Azure Synapse. 
+Before setting up Azure Synapse as a destination in RudderStack, make sure you have an [Azure subscription](https://azure.microsoft.com/en-in/get-started/synapse-analytics/) for the SQL Data Warehouse. 
+
+Additionally, you need a dedicated SQL pool created in Azure Synapse. If you do not have one, you can create it by following the [Azure Synapse documentation](https://docs.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/create-data-warehouse-portal).
```
Comment (ishachopra0401) [inline docs/data-warehouse-integrations/azure-synapse.mdx:27]:
```suggestion
Additionally, you need to create a dedicated SQL pool in Azure Synapse. You can create it by following the [Azure Synapse documentation](https://docs.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/create-data-warehouse-portal).
```

**Review by ishachopra0401 — APPROVED** [review]

File: docs/data-warehouse-integrations/azure-synapse.mdx [hunk 4]
```diff
@@ -20,9 +20,17 @@ Refer to the <a href="https://rudderstack.com/docs/data-warehouse-integrations/w
 Find the open-source transformer code for this destination in the <a href="https://github.com/rudderlabs/rudder-transformer/tree/master/v0/destinations/azure_synapse">GitHub repository</a>.
 </div>
 
-## Setting user permissions in Azure Synapse
+## Prerequisites
 
-To set up an Azure Synapse Analytics user, you require an Azure subscription for the SQL Data Warehouse. Additionally, you need to [create a schema (dedicated SQL pool)](https://docs.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/create-data-warehouse-portal) in Azure Synapse. 
+Before setting up Azure Synapse as a destination in RudderStack, make sure you have an [Azure subscription](https://azure.microsoft.com/en-in/get-started/synapse-analytics/) for the SQL Data Warehouse.
```
Comment (saramashfej) [inline docs/data-warehouse-integrations/azure-synapse.mdx:25]: The link is here is pointed to the wrong place, can we change this to: "Before setting up Azure Synapse as a destination in RudderStack, make sure you have an Azure subscription and that the [Azure Synapse Workspace] (https://docs.microsoft.com/en-in/azure/synapse-analytics/quickstart-create-workspace) is created "

**Review by saramashfej — COMMENTED** [review]
Looks good to me, only had a comment about one step that points to a wrong link

File: docs/data-warehouse-integrations/azure-synapse.mdx (hunk 4 above)
Comment (ameypv-rudder) [inline docs/data-warehouse-integrations/azure-synapse.mdx:25]:
```suggestion
Before setting up Azure Synapse as a destination in RudderStack, make sure you have an Azure subscription and created the [Azure Synapse Workspace](https://docs.microsoft.com/en-in/azure/synapse-analytics/quickstart-create-workspace).
```

**Pull request merged**